docs/docs/jupyter-integration.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ea7576e-1f54-4983-a8d1-1f6c0e2165de",
+   "metadata": {},
+   "source": [
+    "# Jupyter Integration\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5d9b03a-627f-44d1-87da-caf2597f29dd",
+   "metadata": {},
+   "source": [
+    "Datapane has first-class Jupyter Notebook support! This includes Jupyter Lab, Jupyter Notebook, Google Colab, and Visual Studio Code with the Jupyter plugin.\n",
```

Comment:
Stylistically I would have fewer exclamation marks through this page to match the style of the rest of the docs.